main/com.spotify/spotifyd.js

```diff
@@ -100,7 +100,7 @@ module.exports = class SpotifyDaemon {
         switch (process.arch) {
             case 'x64':
                 break;
-            case 'armhf':
+            case 'arm':
```

Comment:
Should we handle both cases, just to be sure? Not sure under which configuration one or the other can be reported.

Reply:
We may later support other arm variants (based on the flags from process.config.variables.arm_*).
The possible values for process.arch are arm and arm64.
With the armhf binary we have support for armv7 and armv8 (both 32 bits).